Provider FAQ Update

Zblocker64 opened the pull request.

Zblocker64 committed:
Squash merge: Provider FAQ and operational docs from PR 631

Zblocker64 committed:
chore: fix extra pages and delete master page

Zblocker64 committed:
chore: fix folder name

Zblocker64 committed:
chore: fix ts

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,330 @@
+
+```
+kubectl -n akash-services get statefulsets
+kubectl -n akash-services get pods -l app=akash-provide
````
Typo

oiclid replied:
fixed.

Zblocker64 replied:
No its not

oiclid replied:
fixed. Turns out I fixed the wrong typo.

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,330 @@
+- Using the example command syntax we will list the last ten entries in Provider logs and enter a live streaming session of new logs generated
+
+```
+kubectl -n akash-services logs $(kubectl -n akash-services get pods -l app=akash-provider --output jsonpath='{.items[-1].metadata.name}') --tail=10 -f
````
you can just run

`kubectl logs akash-provider-0 -n akash-services --tail=50 -f`

oiclid replied:
fixed

Zblocker64 replied:
No its not

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,330 @@
+- The order receipt is one of many event sequences that can be verified within provider logs
+
+```
+kubectl -n akash-services logs $(kubectl -n akash-services get pods -l app=akash-provider --output jsonpath='{.items[-1].metadata.name}') --tail=10 -f
````
This is not at all how logs look now.

Here is how they look.
```
I[2025-04-29|20:35:25.057] order detected                               module=bidengine-service cmp=provider order=order/akash1d2xltxu5vmsxza8gu2j9vudcpacuntn6r72aee/21299713/1/1
I[2025-04-29|20:35:25.058] group fetched                                module=bidengine-order cmp=provider order=akash1d2xltxu5vmsxza8gu2j9vudcpacuntn6r72aee/21299713/1/1
D[2025-04-29|20:35:25.058] unable to fulfill: incompatible attributes for resources requirements module=bidengine-order cmp=provider order=akash1d2xltxu5vmsxza8gu2j9vudcpacuntn6r72aee/21299713/1/1 wanted="{Name:akash Requirements:{SignedBy:{AllOf:[] AnyOf:[]} Attributes:[]} Resources:[{Resources:{ID:1 CPU:units:<val:\"2000\" >  Memory:quantity:<val:\"10737418240\" >  Storage:[{Name:default Quantity:{Val:32212254720} Attributes:[]}] GPU:units:<val:\"1\" > attributes:<key:\"vendor/nvidia/model/rtx-3060-ti\" value:\"true\" >  Endpoints:[{Kind:RANDOM_PORT SequenceNumber:0}]} Count:1 Price:30000.000000000000000000uakt}]}" have="[{Key:host Value:akash} {Key:tier Value:community} {Key:organization Value:Akash Provider Services} {Key:email Value:info@akashprovid.com} {Key:status-page Value:status.akashprovid.com} {Key:location-region Value:na-us-southwest} {Key:country Value:US} {Key:city Value:DAL} {Key:timezone Value:utc-5} {Key:location-type Value:home} {Key:capabilities/cpu Value:amd} {Key:capabilities/cpu/arch Value:x86-64} {Key:capabilities/gpu Value:nvidia} {Key:capabilities/storage/1/class Value:beta2} {Key:capabilities/storage/1/persistent Value:false} {Key:capabilities/memory Value:ddr4ecc} {Key:network-provider Value:Pavlov Media} {Key:network-speed-up Value:1000} {Key:network-speed-down Value:1000} {Key:capabilities/storage/2/class Value:beta3} {Key:capabilities/storage/2/persistent Value:true} {Key:feat-persistent-storage Value:true} {Key:capabilities/gpu/vendor/nvidia/model/rtx4090 Value:true} {Key:capabilities/gpu/vendor/nvidia/model/rtx4090/ram/24Gi Value:true} {Key:capabilities/gpu/vendor/nvidia/model/rtx4090/ram/24Gi/interface/pcie Value:true} {Key:capabilities/gpu/vendor/nvidia/model/rtx4090/interface/pcie Value:true} {Key:capabilities/storage/3/class Value:ram} {Key:capabilities/storage/3/persistent Value:false} {Key:cuda Value:12.7}]"
D[2025-04-29|20:35:25.059] declined to bid                              module=bidengine-order cmp=provider order=akash1d2xltxu5vmsxza8gu2j9vudcpacuntn6r72aee/21299713/1/1
I[2025-04-29|20:35:25.059] shutting down                                module=bidengine-order cmp=provider order=akash1d2xltxu5vmsxza8gu2j9vudcpacuntn6r72aee/21299713/1/1
D[2025-04-29|20:36:00.041] cluster resources dump={"nodes":[{"name":"4090-node","allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502},"available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526}}],"total_allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502,"storage":{"beta3":2445568376832}},"total_available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526,"storage":{"beta3":2179280404480}}} module=provider-cluster cmp=provider cmp=service cmp=inventory-service
D[2025-04-29|20:37:00.839] cluster resources dump={"nodes":[{"name":"4090-node","allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502},"available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526}}],"total_allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502,"storage":{"beta3":2445568376832}},"total_available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526,"storage":{"beta3":2179280404480}}} module=provider-cluster cmp=provider cmp=service cmp=inventory-service
D[2025-04-29|20:38:01.687] cluster resources dump={"nodes":[{"name":"4090-node","allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502},"available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526}}],"total_allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502,"storage":{"beta3":2445568376832}},"total_available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526,"storage":{"beta3":2179280404480}}} module=provider-cluster cmp=provider cmp=service cmp=inventory-service
2025-04-29 20:38:44.338894 I | http: TLS handshake error from 172.16.224.203:54788: EOF
D[2025-04-29|20:39:02.531] cluster resources dump={"nodes":[{"name":"4090-node","allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502},"available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526}}],"total_allocatable":{"cpu":224000,"gpu":8,"memory":506969817088,"storage_ephemeral":1868399843502,"storage":{"beta3":2445568376832}},"total_available":{"cpu":129690,"gpu":5,"memory":351501647872,"storage_ephemeral":1430067812526,"storage":{"beta3":2179280404480}}} module=provider-cluster cmp=provider cmp=service cmp=inventory-service
```

oiclid replied:
fixed

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
```diff
@@ -0,0 +1,330 @@
+
+> Conduct these steps on each Akash Provider/Kubernetes worker node that hosts GPU resources unless stated otherwise.
+
+### Basic GPU Resource Verifications
```
I think all you need is nvidia-smi

oiclid replied:
so, do we not need to install `torch` and `numpy`?

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,330 @@
+```
+
+### Ensure Correct Version/Presence of NVIDIA Device Plugin
+
````
Need to fix this

oiclid replied:
It was fixed in the original docs (a few lines below). It seems the original writer forgot to delete this.

Zblocker64 replied:
Then lets delete it

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
```diff
@@ -0,0 +1,330 @@
+
+### NVIDIA Fabric Manager
+
+TBD
```
Need to fix this

oiclid replied:
same as above.

Zblocker64 replied:
Then lets delete it

oiclid replied:
I deleted it. I can't merge into your branch because I don't have the right to do so. But it's been fixed in the PR I shared. Can you give me write access?

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,330 @@
+[Thu Sep 28 19:29:05 2023] nvidia-uvm: Loaded the UVM driver, major device number 235.
+```
+
+### Ensure Correct Version/Presence of NVIDIA Device Plugin
````
Better to check pods, helm could be deployed but pods may not be running

oiclid replied:
fixed

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
```diff
@@ -0,0 +1,330 @@
+
+### NVIDIA Fabric Manager
+
+- In certain configurations (e.g., non-PCIe GPU setups like SXM form factors), installing the **NVIDIA Fabric Manager** on worker nodes hosting GPU resources may be necessary.
```
This is for all non PCIE

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
```diff
@@ -0,0 +1,330 @@
+
+Further details: [NVIDIA Developer Forum Thread](https://forums.developer.nvidia.com/t/error-802-system-not-yet-initialized-cuda-11-3/234955)
+
+> _**NOTE**_ - Replace `525` with your installed NVIDIA driver version.
```
default needs to be 565

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
```diff
@@ -0,0 +1,330 @@
+
+#### Fix: Use Official NVIDIA Repository
+
+> _**NOTE**_ - Replace `2204` with your Ubuntu version (e.g., `2404` for Ubuntu 24.04).
```
Default needs to be ubuntu 24.04

oiclid replied:
fixed

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/lease-management/index.md:
```diff
@@ -1582,15 +1083,15 @@ A common symptom of this issue is that **nvidia-fabricmanager** fails to start p
 
 To resolve this issue, you’ll need to use the official NVIDIA repository. Here's how to add it:
 
 > _**NOTE**_ - replace `2204` with your Ubuntu version (e.g. `2404` for Ubuntu noble release)
```
24.04 should be default

oiclid replied:
fixed

oiclid commented:
PR is [here](https://github.com/akash-network/website/pull/634)

oiclid committed:
chore: competing edits for Provider FAQs Docs. (#634)

1. Fixed all the issues raised by the reviewer on the file s`src/content/Docs/providers/providers-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md` and  `src/content/Docs/providers/providers-faq-and-guide/lease-management/index.md`

Zblocker64 commented on src/content/Docs/providers/provider-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md:
````diff
@@ -0,0 +1,323 @@
+> _**NOTE**_ - This method is prefered as helm could be deployed but pods may not be running
+
+```bash
+kubectl get pods -n kube-system -l name=nvidia-device-plugin-ds
````
This is not the correct namespace

oiclid replied:
let me look at the docs again.

oiclid replied:
I think i fixed it now.

oiclid commented:
PR [here](https://github.com/akash-network/website/pull/635)

oiclid committed:
chore: fixed further issues in Provider FAQ docs (#635)

* chore: competing edits for Provider FAQs Docs.

1. Fixed all the issues raised by the reviewer on the file s`src/content/Docs/providers/providers-faq-and-guide/Maintenance-Logs-and-Troubleshooting/index.md` and  `src/content/Docs/providers/providers-faq-and-guide/lease-management/index.md`

* chore: fixed further issues in Provider FAQ docs
